modules/administration-guide/partials/assembly_authorizing-users.adoc

```diff
@@ -33,6 +33,4 @@ include::partial$proc_listing-che-permissions.adoc[leveloffset=+1]
 
 include::partial$proc_assigning-che-permissions.adoc[leveloffset=+1]
 
-include::partial$proc_sharing-che-permissions.adoc[leveloffset=+1]
```

Comment:
@themr0c Two more questions:
1. Since the proc_ file (**Share** button) is removed, I cannot find an admin procedure that explains the admin how to invited a user to a workspace. Can you?
2. Is a workspace still shared/sharable with a user (for example, as a result of another procedure or in a concept module) despite the absence of the **Share** button? If so, it'd be useful to check that other procedure or concept module if sharing the workspace is mentioned in it.